doc/client-howto.md

```diff
@@ -141,18 +159,22 @@ but will not be able to retrieve any files.
 
 * `KACHERY_STORAGE_DIR`: If set, this defines the directory in
 which the client will look for files locally before attempting
-to obtain them from the node. QUERY: The environment variable
-will be honored over the config file.
+to obtain them from the node. It must coincide with the $KACHERY_STORAGE_DIR
+for the running daemon. If the daemon is running on a different
+computer it is okay if the literal value of $KACHERY_STORAGE_DIR
+is different as long as it is referring to the same underlying
+directory.
```

Comment:
That won't work because the client looks for a special auth code in the storage directory which it needs to send along with every request to the daemon proving that it has permission to access that auth file. So if you tried to misconfigure it, then the client would complain.